tests/unit/FileExports/CustomerFileExportTest.php

```diff
@@ -14,6 +17,51 @@ public function getFileExportClass(): string
         return CustomerFileExport::class;
     }
 
+    public function dataProviderUserCreateByRoles()
+    {
+        $tests = [];
+        foreach (GOCustomer::VALID_ROLES as $role) {
+            $tests['valid-'.$role] = [$role, $role.'_test'];
```

Comment:
you can just put  `_test` inside the test
```suggestion
            $tests['valid-'.$role] = [$role];
```